test/suites/alias.sh

```diff
@@ -0,0 +1,143 @@
+test_alias() {
+  ensure_import_testimage
+
+  echo "Test 1: Add a new alias"
+  lxc alias add my-list "list -c ns46S"
+  lxc alias list --format csv | grep  -xF "my-list,list -c ns46S"
+
+  echo "Test 2: Prevent adding duplicate alias (should fail)"
+  ALIAS_ERR="$(! lxc alias add my-list "list -c -s" 2>&1 || echo fail)"
+  [ "$(echo "${ALIAS_ERR}" | tail -1)" = "Error: Alias my-list already exists" ]
+
+  echo "Test 3: Show alias in YAML format"
+  lxc alias show | grep -xF "my-list: list -c ns46S"
+
+  echo "Test 4: Rename an alias"
+  lxc alias rename my-list my-list2
+  ! lxc alias list --format csv | grep -xF "my-list," || false
+  lxc alias list --format csv | grep -xF "my-list2,list -c ns46S"
+
+  echo "Test 5: Prevent rename to existing alias (should fail)"
+  lxc alias add another-alias "list -c s"
+  ALIAS_ERR="$(! lxc alias rename another-alias my-list2 2>&1 || echo fail)"
+  [ "$(echo "${ALIAS_ERR}" | tail -1)" = "Error: Alias my-list2 already exists" ]
```

Comment:
Oh I didn't know there was a header/line to drop. LGTM as-is but the `grep` also made sense because of that, sorry.